backend/app/routes/upload_config_router.py

```diff
@@ -106,24 +105,20 @@ async def extract_config(
     return JSONResponse(status_code=status.HTTP_200_OK, content=response)
 
 
-@router.post("/configure-devices/{lab_group_number}")
+@router.post("/configure_devices/{lab_group_number}/{topology_id}")
```

Comment:
{}{} odwrotnie wygląda lepiej jak już lubisz kosmetykę

Reply:
Jak i tak coś będę poprawiał to to też zmienię